project.clj

```diff
@@ -1,13 +1,12 @@
-(defproject clojurewerkz/cassaforte "3.0.0-alpha2-SNAPSHOT"
```

Comment:
Never change version number in pull requests. It's up to the maintainer to decide when to do it.